firebase-project/firestore.rules

```diff
@@ -0,0 +1,9 @@
+rules_version = '2';
+service cloud.firestore {
+  match /databases/{database}/documents {
+    match /{document=**} {
+      allow get, write: if true;
+      allow list: if false;
```

Comment:
Can you add a comment just before to never set it to true, otherwise it would allow people to list all the scenes and overwrite them with garbage (they won't be able to read it though, thanks end to end encryption).